pom.xml

```diff
@@ -21,7 +21,7 @@
     <groupId>org.wso2.integration.connector</groupId>
     <artifactId>mi-connector-gmail</artifactId>
     <packaging>jar</packaging>
-    <version>3.0.10-SNAPSHOT</version>
+    <version>3.0.9-SNAPSHOT</version>
```

Comment:
Is this intentional?

Reply:
Yes. When we triggered earlier the version got upgraded but the release was unsuccessful.